pygmt/gridops.py

```diff
@@ -20,13 +20,15 @@
 
 @fmt_docstring
 @use_alias(
```

Comment:
You're changing the alias settings for `grdcut`. That's incorrect. You should revert the changes here and add decorators `@fmt_docstring`, `@use_alias` and `@kwargs_to_strings` just before the `grdfilter` function, i.e., each function has its own decorators.

Reply:
Ok.. I thought they could be the same. Thanks!